add getApipData to itemRunner - Acces the data about APIP the content and related media files

lecosson opened the pull request:
Related to: https://oat-sa.atlassian.net/browse/TAO-9637
Acces the data about APIP the content and related media files

update packages versions
#### Dependencies
Requires :
 - [ ] https://github.com/oat-sa/tao-item-runner-qti-fe/pull/34

lecosson committed:
add getApipData to itemRunner - install changed branch instead of current version
https://oat-sa.atlassian.net/browse/TAO-9637 ([ACT] [TTS - FE] Acces the data about APIP the content and related media files)

jsconan reviewed (changes requested):
The target link in `package.json` is ok.
But why did you remove the `package-lock.json` file???
It must be versioned.

When working with git branch instead of version, we must only touch the `package.json`.
And the `package-lock.json` file must be updated only once a proper version is available and set in the `package.json`.

lecosson committed:
restore package-lock.json file and bump version
https://oat-sa.atlassian.net/browse/TAO-9637 ([ACT] [TTS - FE] Acces the data about APIP the content and related media files)

lecosson committed:
restore package-lock.json file and bump version
https://oat-sa.atlassian.net/browse/TAO-9637 ([ACT] [TTS - FE] Acces the data about APIP the content and related media files)

atsymuk committed:
Merge remote-tracking branch 'origin/develop' into feature/TAO-9637/ACT_TTS_FE_Acces_the_data_about_APIP_the_content_and_related_media_files_v2
